tiatoolbox/utils/misc.py

```diff
@@ -1197,24 +1199,34 @@ def add_from_dat(
     store.append_many(anns)
 
 
-def patch_pred_store(
+def dict_to_store(
     patch_output: dict,
     scale_factor: tuple[int, int],
     class_dict: dict | None = None,
-) -> AnnotationStore:
+    save_path: Path | None = None,
+) -> AnnotationStore | Path:
     """Create an SQLiteStore containing Annotations for each patch.
 
```

Comment:
Please add in the description that this function takes the output of TIAToolbox engines and converts these to AnnotationStore.